flink-runtime/src/main/java/org/apache/flink/runtime/instance/SlotSharingGroupAssignment.java

```diff
@@ -106,7 +106,7 @@
 
 	/**
 	 * Gets the number of slots that are currently governed by this assignment group.
-	 * This refers to the slots allocated from an {@link org.apache.flink.runtime.instance.Instance},
+	 * This refers to the slots allocated from an Instance,
```

Comment:
Temporarily because this file/class is also of legacy code and tracked by FLINK-13334